exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,17 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+function greetings(mentor) {
+    let mentor1 = "Daniel";
```

Comment:
The variables `mentor1`, `mentor2`, `mentor3`, `mentor4` and `mentor5` are not used anywhere. You shouldn't leave unused variables in the code.